### PR TITLE
Fix #85 added empty background

### LINKED ISSUE #85: Allow to print on empty background
## Description
Actually empty background is not selectable thus we cannot use it to print the map.

## Depends on
- [2382](https://github.com/geosolutions-it/MapStore2/issues/2382)

## Todo
- [ ] Make it selectable
- [ ] Verify that print plugin works as expected.

### PR DESCRIPTION
**Pull request by MV88**
Fix #85 

name was removed in order to render a gray background 
image

### CODE REVIEW

File: new.json [hunk 1]
```diff
@@ -78,14 +74,15 @@
...
         "title": "Nessuno sfondo",
         "fixed": true,
-        "type": "OpenLayers.Layer",
-        "visibility": false,
+        "type": "ol",
+        "visibility": true,
         "args": [
           "Nessuno sfondo", {
-            "visibility": false
+            "visibility": true
```
Comment (offtherailz) [inline new.json]: Why visibility = true. This may force the layer to be selected by default

File: new.json [hunk 2]
```diff
@@ -78,14 +74,15 @@
       },
       {
         "source": "ol",
+        "name": "Nessuno sfondo",
         "group": "background",
         "title": "Nessuno sfondo",
         "fixed": true,
-        "type": "OpenLayers.Layer",
```
Comment (offtherailz) [inline new.json]: There were old maps with empty background with this type. Are they selectable?

Comment (MV88) [reply new.json]: for this we need to make a change [here](https://github.com/geosolutions-it/MapStore2/blob/master/web/client/utils/LayersUtils.js#L68)

File: new.json (hunk 1 above)
Comment (MV88) [reply new.json]: going to restore this value to false

**Pull request merged**